### ISSUE TITLE
Wrong message format

### ISSUE THREAD

**Issue by JaCoLJcL** [ivanmihov/homeassistant-ntfy.sh#6]
He, great addon, but whem I send a TEST msg from HA to my chanel I got this creepy stuf:

![image](https://github.com/ivanmihov/homeassistant-ntfy.sh/assets/70702183/bdae5217-417a-445f-bf09-aa47a9a5d749)

How to "convert" this to normal TEST message ?

**Comment by arafalko** [comment]
Hello,
I faced the same today. Problem on my side was I put topic in the url to ntfy.sh. From ntfy docs sending json payload should be done to the root of the app and topic should be set in it.

**Comment by albumihai** [comment]
> Hello, I faced the same today. Problem on my side was I put topic in the url to ntfy.sh. From ntfy docs sending json payload should be done to the root of the app and topic should be set in it.

Yes , you  are right, if you only use the url without the topic it works as it should. Thanks for you tip!
